docs/caching/includes/redis-app-host.md

```diff
@@ -145,3 +145,40 @@ builder.AddProject<Projects.ExampleProject>()
 ```
 
 The preceding code adds persistence to the Redis resource by taking snapshots of the Redis data at a specified interval and threshold. The `interval` is time between snapshot exports and the `keysChangedThreshold` is the number of key change operations required to trigger a snapshot. For more information on persistence, see [Redis docs: Persistence](https://redis.io/topics/persistence).
+
+### Azure Redis hosting integration
+
+To deploy your Redis resources to Azure, install the [📦 Aspire.Hosting.Azure.Redis](https://www.nuget.org/packages/Aspire.Hosting.Azure.Redis) NuGet package:
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Azure.Redis
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Aspire.Hosting.Azure.Redis"
+                  Version="*" />
+```
+
+---
+
+#### Add Azure Redis server resource
+
+After you've installed the .NET Aspire hosting Azure Redis package, call the `AddAzureRedis` extension method in your app host project:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var cache = builder.AddAzureRedis("azcache")
+
+var exampleProject = builder.AddProject<Projects.ExampleProject>()
+                            .WithReference(cache);
+```
+
+The preceding call to `AddAzureRedis` configures the Redis server resource to be deployed as an [Azure Cache for Redis](/azure/azure-cache-for-redis/cache-overview).
+
+> [!IMPORTANT]
+> By default, `AddAzureRedis` configures [Microsoft Entra ID](/azure/postgresql/flexible-server/concepts-azure-ad-authentication) authentication. This requires changes to applications that need to connect to these resources. For more information, see [Client integration](#client-integration).
```

Comment:
```suggestion
> By default, `AddAzureRedis` configures [Microsoft Entra ID](azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication) authentication. This requires changes to applications that need to connect to these resources. For more information, see [Client integration](#client-integration).
```

It points to PG docs on Entra, but I'm assuming that is just a copy/paste mistake.